src/ol/layer/WebGLPoints.js

```diff
@@ -9,7 +9,7 @@ import {parseLiteralStyle} from '../webgl/ShaderBuilder.js';
 /**
  * @template {import("../source/Vector.js").default<import("../geom/Point.js").default>} VectorSourceType
  * @typedef {Object} Options
- * @property {import('../style/literal.js').LiteralStyle} style Literal style to apply to the layer features.
+ * @property {LiteralStyle} style Literal style to apply to the layer features.
```

Comment:
Apidoc generation failed with an error without this change presumable because without any exports `ol/style/literal.js` is not a module. Though with this change no apidocs are generated for the typedefs in there.
Maybe the file content could be added somewhere else?